bamboo/unit-tests.sh

```diff
@@ -3,7 +3,7 @@ set -ex
 error_to_s3 () {
   ngap_env=$(echo $NGAP_ENV | awk '{print tolower($0)}')
   bucket_name=${ngap_env}-unit-test-error-logs
-  aws s3api create-bucket --bucket $bucket_name | echo "bucket ${bucket_name} could not be created"
+  aws s3api create-bucket --bucket $bucket_name 2>/dev/null | echo "bucket ${bucket_name} could not be created"
```

Comment:
I grok what this was doing, but why was this change needed?       If we pipe the output to /dev/null and it fails, we won't know why, right?

Reply:
this is something I can leave behind cause it doesn't matter but
it's *going* to fail (nearly) every time because the bucket probably already exists.
so we use the | case to keep it from interrupting anything and swallow it's unhelpful "already exists" error
I just got tired of seeing it tell me this was an error

Reply:
Ah I see - this isn't too big a deal to leave in given it's a helper.  If we really need diagnostics we can always update it.